R/expect-all.R

```diff
@@ -0,0 +1,67 @@
+#' Do you expect every value in a vector to have this value?
```

Comment:
```suggestion
#' Does a vector have a constant value?
```

Just thinking out loud